README.md

```diff
@@ -1 +1,60 @@
-# kex
\ No newline at end of file
+The purpose of kex is to provide an opinionated executor into a kubernetes container.  Traditionally, if one wanted to exec into a kubernetes container, one had to `kubectl get pods --namespace foo`, visually identify the pod of interest, copy that pod to the buffer, and then `kubectl --namespace foo exec -it <paste_buffer> bash` to exec in to the pod.  This simple utility aims to provide a namespace-specific pod selector for quick execution.
+
+# kex
+
+kex is a pod exec utility.
+
+```
+USAGE:
+  kex             : exec in to the default pod
+  kex ls,list     : list the available pods
+  kex <NUM>       : exec into a specified pod
+  kex -h,--help   : show this message
+```
+
+### USAGE
+
+```sh
+$ kex ls
+NAME                                      SELECTOR
+365bass-dt1nb                             1 [default]
```

Comment:
I just meant for if naming privacy mattered (thinking it may have been an initial sketch only) - cool 👍 . I'm with you, no attack concern 💥